crates/bevy_render/src/primitives/mod.rs

```diff
@@ -4,6 +4,18 @@ use bevy_reflect::Reflect;
 use bevy_utils::HashMap;
 
 /// An axis-aligned bounding box.
+///
+/// It represents a box covering the local space occupied by the entity, with faces
+/// orthogonal to the local axis. It is used as a component on an entity to determine
+/// if it should be rendered by a [`Camera`](crate::camera::Camera) entity if it
+/// intersects with its [`Frustum`], in a process called frustum culling.
```

Comment:
saying it's used for frustum culling is very limiting to `Aabb`, which is why it's in the primitive module and not in view. `Aabb` are often used for collisions, for debug, for sorting in space, ...

Reply:
Is it really? Because it's not inserted to entities with `NoFrustumCulling`, so if it is, shouldn't `NoFrustumCulling` be handled differently?

Reply:
In Bevy it's used just for frustum culling, but an aabb is a general type that have many uses. It's a component that users can use however they want, we shouldn't restrict it to frustum culling in its doc

Reply:
I agree, but since it's linked to `NoFrustumCulling`, I don't see it being used for something else.

Reply:
Addressed partially by https://github.com/bevyengine/bevy/pull/9136/commits/936a9122b44575c948e59c443c37a13bbc9d01c6